.rubocop.yml

```diff
@@ -65,6 +68,7 @@ Style/Documentation:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
+
```

Comment:
Mind checking this file? It seems like it has a lot of extra blank lines.

Reply:
I added 2 empty lines on purpose, to identify easier the 3 sections: ruby, rails and rspec 👀 🙈

Reply:
Can't each gem read its own file?

Reply:
I think that there's a way to do that 🤔  I'll try and let you know what happens